README.md

```diff
@@ -1,13 +1,26 @@
-# dave
+# Dave
+
 Dave cuts VoIPGRID releases.
 
-## Installation
+## Status
+
+Maintained
+
+## Usage
+
+### Requirements
+
+You'll need a Github access token, which you can generate
+[here](https://github.com/settings/tokens). Dave only needs `repo` scope to
+work.
+
+### Installation
 
 Quite simple: get [Go](https://golang.org/dl), set a `$GOPATH`, and run
 
     go get github.com/VoIPGRID/dave
 
-## Use
+### Running
```

Comment:
This is the default template we use. If I where to change it here I need to ask 10 other projects to change theirs as well. This Usage and nesting is based on research of JJ